### ISSUE TITLE
Investigate integration with EggTimer, a menstrual cycle tracker

### ISSUE THREAD

**Issue by patcon** [HumanDynamics/openPDS#44]
@jessamynsmith is working on an open source menstrual tracker. It strikes me as a great opportunity to make use of Open PDS's model, as this is such personal information.

* https://github.com/jessamynsmith/eggtimer-server
* https://github.com/jessamynsmith/eggtimer-ui

Anyone have any thoughts on whether this is a good application? I'm relatively new to OpenPDS :)

(As I understand it, it's being built with cordova, so the intention is to package it as a cross-platform app. Correct me if I'm wrong though, Jessamyn!)

**Comment by jessamynsmith** [comment]
Yes, my back end application (eggtimer-server) is in Django, and the long-term goal is to port the UI to the Angular app (eggtimer-ui) so the Django server is basically providing an API.
